pkg/protocol/faststate/machine_test.go

```diff
@@ -0,0 +1,232 @@
+package faststate
+
+import (
+	"context"
+	"encoding/json"
+	"math/rand"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/keep-network/keep-core/pkg/internal/testutils"
+	"github.com/keep-network/keep-core/pkg/net"
+	netLocal "github.com/keep-network/keep-core/pkg/net/local"
```

Comment:
According to Go's convention about package names/aliases (https://go.dev/blog/package-names) we should use lowercase here:
```suggestion
	netlocal "github.com/keep-network/keep-core/pkg/net/local"
```

Reply:
d695c91cc64f4db2831e26e8ba8f3a87712d9dd7